packages/design-system-twrnc-preset/src/twrnc-settings/typography.ts

```diff
@@ -77,7 +77,7 @@ export const typographyTailwindConfig: TypographyTailwindConfigProps = {
   },
   fontFamily: {
     sans: [
-      'EuclidCircularB-Regular',
+      'CentraNo1-Book',
```

Comment:
Not sure if we can should be using the `fontFamilies` object from the design tokens here? There seems to be a difference between what is here and what is in the object so I'll leave it for the time being.

https://github.com/MetaMask/metamask-design-system/pull/499/files#diff-9c75dc4f659e3ef07dc5cba9bb52805cac1e2cf11330f9daaa60b5d146783f9bL2